src/modules/simulator/simulator_mavlink.cpp

```diff
@@ -1186,8 +1183,7 @@ int Simulator::publish_distance_topic(mavlink_distance_sensor_t *dist_mavlink)
 {
 	uint64_t timestamp = hrt_absolute_time();
 
-	struct distance_sensor_s dist;
-	memset(&dist, 0, sizeof(dist));
+	struct distance_sensor_s dist = {};
```

Comment:
In c++ you can drop the "struct".

Reply:
Oh true, it was only C cases I ran against before with PX4 in which it broke the build... I'll take it into account from now on, thanks 👍